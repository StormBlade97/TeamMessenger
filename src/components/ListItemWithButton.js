import React from 'react'

const ListItemWithButton = (props) => {

    return (
        <li>
            {props.itemText} <button onClick={props.onButtonClick}>{props.buttonText}</button>
        </li>
    )
}

export default ListItemWithButton