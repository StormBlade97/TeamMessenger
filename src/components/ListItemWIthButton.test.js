import React from 'react'
import ListItemWithButton from './ListItemWithButton'
import renderer from 'react-test-renderer'

describe('Component: ListItemWithButton', () => {

    it('renders a snapshot', () => {
        const tree = renderer.create(
            <ListItemWithButton
                itemText="Bob"
                buttonText="remove"
                onButtonClick={() => {}}/>
        ).toJSON()
        expect(tree).toMatchSnapshot()
    })
})