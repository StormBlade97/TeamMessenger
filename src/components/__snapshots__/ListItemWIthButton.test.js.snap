exports[`Component: ListItemWithButton renders a snapshot 1`] = `
<li>
  Bob
   
  <button
    onClick={[Function]}>
    remove
  </button>
</li>
`;
